docs/tutorials/11_quadratic_hamiltonian_and_slater_determinants.ipynb

```diff
@@ -0,0 +1,440 @@
+    "A quadratic Hamiltonian is initialized like so:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "6b91735f-72e4-46de-a2fa-d2d89db53d6a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Fermionic Operator\n",
+      "register length=4, number terms=29\n",
+      "  4.0\n",
+      "+ 1.0 * ( +_0 -_0 )\n",
+      "+ 2.0 * ( +_0 -_1 )\n",
+      "+ 2.0 * ( +_1 -_0 )\n",
+      "+ 3.0 * ( +_0 +_1 )\n",
+      "+ 3.0 * ( -_1 -_0 )\n",
+      "+ 0.0 * ( +_0 -_2 )\n",
+      "+ 0.0 * ( +_2 -_0 )\n",
+      "+ 0.0 * ( +_0 +_2 )\n",
+      "+ 0.0 * ( -_2 -_0 )\n",
+      "+ 0.0 * ( +_0 -_3 )\n",
+      "+ 0.0 * ( +_3 -_0 )\n",
+      "+ 0.0 * ( +_0 +_3 )\n",
+      "+ 0.0 * ( -_3 -_0 )\n",
+      "+ 1.0 * ( +_1 -_1 )\n",
+      "+ 2.0 * ( +_1 -_2 )\n",
+      "+ 2.0 * ( +_2 -_1 )\n",
+      "+ 3.0 * ( +_1 +_2 )\n",
+      "+ 3.0 * ( -_2 -_1 )\n",
+      "+ 0.0 * ( +_1 -_3 )\n",
+      "+ 0.0 * ( +_3 -_1 )\n",
+      "+ 0.0 * ( +_1 +_3 )\n",
+      "+ 0.0 * ( -_3 -_1 )\n",
+      "+ 1.0 * ( +_2 -_2 )\n",
+      "+ 2.0 * ( +_2 -_3 )\n",
+      "+ 2.0 * ( +_3 -_2 )\n",
+      "+ 3.0 * ( +_2 +_3 )\n",
+      "+ 3.0 * ( -_3 -_2 )\n",
+      "+ 1.0 * ( +_3 -_3 )\n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "from qiskit_nature.operators.second_quantization import QuadraticHamiltonian\n",
+    "\n",
+    "# create Hamiltonian\n",
+    "hermitian_part = np.array(\n",
+    "    [\n",
+    "        [1.0, 2.0, 0.0, 0.0],\n",
+    "        [2.0, 1.0, 2.0, 0.0],\n",
+    "        [0.0, 2.0, 1.0, 2.0],\n",
+    "        [0.0, 0.0, 2.0, 1.0],\n",
+    "    ]\n",
+    ")\n",
+    "antisymmetric_part = np.array(\n",
+    "    [\n",
+    "        [0.0, 3.0, 0.0, 0.0],\n",
+    "        [-3.0, 0.0, 3.0, 0.0],\n",
+    "        [0.0, -3.0, 0.0, 3.0],\n",
+    "        [0.0, 0.0, -3.0, 0.0],\n",
+    "    ]\n",
+    ")\n",
+    "constant = 4.0\n",
+    "\n",
+    "hamiltonian = QuadraticHamiltonian(\n",
+    "    hermitian_part=hermitian_part,\n",
+    "    antisymmetric_part=antisymmetric_part,\n",
+    "    constant=constant,\n",
+    ")\n",
+    "\n",
+    "# convert it to a FermionicOp and print it\n",
+    "hamiltonian_ferm = hamiltonian.to_fermionic_op()\n",
+    "hamiltonian_ferm.set_truncation(0)\n",
+    "print(hamiltonian_ferm)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93b2ca42-9947-45da-ad1e-7c11735dc5a5",
+   "metadata": {},
+   "source": [
+    "## Diagonalization and state preparation\n",
+    "A quadratic Hamiltonian can always be rewritten in the form\n",
+    "$$\n",
+    "    H = \\sum_{j} \\varepsilon_j b^\\dagger_j b_j + \\text{constant}\n",
+    "$$\n",
+    "where $\\varepsilon_0 \\leq \\varepsilon_1 \\leq \\cdots \\leq \\varepsilon_N$ are non-negative real numbers called orbitals energies and\n",
+    "the $\\{b_j^\\dagger\\}$ are a new set of fermionic creation operators\n",
```

Comment:
And analogously here, you should also include the `b_j` annihilation operators.

Reply:
Sorry, I'm not sure how you would like it to be worded. The current wording seems perfect to me. If you insist on changing it, please make an explicit suggestion. You can also commit it yourself before merging.

Reply:
```suggestion
    "the $\\{b_j, b_j^\\dagger\\}$ are a new set of fermionic annihilation and creation operators\n",
```